webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -372,6 +374,26 @@ const AdvanceTextEditor = ({
         }
     }, [handleWidthChange, message]);
 
+    const wasNotifiedOfLogIn = LocalStorageStore.getWasNotifiedOfLogIn();
+
+    const ariaLabel = useMemo(() => {
+        let label;
+        if (!wasNotifiedOfLogIn) {
+            label = Utils.localizeMessage(
+                'channelView.login.successfull',
+                'Login Successful',
+            );
+
+            // set timeout to make sure aria-label is read by a screen reader,
+            // and then set the flag to "true" to make sure it's not read again until a user logs back in
+            const timeoutId = setTimeout(() => {
+                LocalStorageStore.setWasNotifiedOfLogIn(true);
+                clearTimeout(timeoutId);
```

Comment:
I am not sure if the clearTimeout actually works using it this way. If the intention is preventing memory leaks for the timeout reference the best way is to call it from the componentWillUnmount (class based) / useEffect return (functional components).

Reply:
I think it does work this way, but via useEffect it looks better, I admit 👍